getting started post (#339)
tailscale-dev/tailscale-dev: data/blog/get-started-in-10-nov2023.mdx

---
title: 'Getting Started with Tailscale in 10 Minutes'
date: 2023-11-22
authors: ['alexktz']
summary: 'Everything you need to know to get started with Tailscale in under 10 minutes.'
tags:
  - Homelab
  - Networking
images: ['/images/getting-started/social-nov2023.jpeg']
---

> Life is better when you connect more devices to your Tailnet - the name we give to the group of devices which forms your Tailscale network. In today's video we cover setting up a Tailnet from scratch and show you how to add your first few devices.

Reviewer: ```suggestion
Life is better when you connect more devices to your tailnet — the group of devices that forms your Tailscale network. In today's video we cover setting up a tailnet from scratch and show you how to add your first few devices.
```

<figure>
  <iframe
    width="560"
    height="315"
    src="https://www.youtube.com/embed/sPdvyR7bLqI?si=aUDgmiwOMUPNNOKu"
    title="YouTube video player"
    frameBorder="0"
    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
    allowFullScreen
  ></iframe>
</figure>

> We also cover the basics of [Tailscale SSH](https://tailscale.com/tailscale-ssh/), [MagicDNS](https://tailscale.com/kb/1081/magicdns/), unique private device IPs, and help you wrap your brain around [exit nodes](https://tailscale.com/kb/1103/exit-nodes/?q=exit%20). Of course, this video just scratches the surface of what's possible with Tailscale - learn more over on our [documentation](https://tailscale.com/kb/1017/install/).

Reviewer: ```suggestion
We also cover the basics of [Tailscale SSH](https://tailscale.com/tailscale-ssh/), [MagicDNS](https://tailscale.com/kb/1081/magicdns/), unique private device IPs, and help you wrap your brain around [exit nodes](https://tailscale.com/kb/1103/exit-nodes/). Of course, this video just scratches the surface of what's possible with Tailscale — learn more over on our [documentation](https://tailscale.com/kb/1017/install/).
```
